DataFormats/JetMatching/interface/JetFlavourInfoMatching.h

```diff
@@ -4,7 +4,7 @@
 #include "DataFormats/JetReco/interface/CaloJetCollection.h"
 #include "DataFormats/Common/interface/AssociationVector.h"
 #include "DataFormats/Common/interface/RefToBaseProd.h"
-#include "SimDataFormats/JetMatching/interface/JetFlavourInfo.h"
+#include "DataFormats/JetMatching/interface/JetFlavourInfo.h"
```

Comment:
it would be nice to update the include guards as well, `s/SimDataFormats_/DataFormats_/` (this applies to all files in this moved package)

Reply:
Sorry I missed those. Fixed.